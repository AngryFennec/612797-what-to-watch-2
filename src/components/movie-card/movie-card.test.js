import React from 'react';
import MovieCard from './movie-card.jsx';
import renderer from 'react-test-renderer';

it(`MovieCard component renders correctly`, () => {
  const tree = renderer
    .create(<MovieCard
      title = {`title`}
      id = {`0`}
      onTitleClick = {jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
