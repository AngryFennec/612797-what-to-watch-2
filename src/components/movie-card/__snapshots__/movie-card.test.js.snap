// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard component renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={0}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height={175}
      muted={true}
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="#"
    />
  </h3>
</article>
`;
